### PR TITLE
fix typo in README

### PR DESCRIPTION
**Pull request by jameslamb**
## Description

Fixes a typo in the README... `consistentcy -> consistency`.

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/rapids-cmake/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The `cmake-format.json` is up to date with these changes.
- [ ] I have added new files under rapids-cmake/
   - [ ] I have added include guards (`include_guard(GLOBAL)`)
   - [ ] I have added the associated docs/ rst file and update the api.rst

### CODE REVIEW

**Comment by jameslamb** [comment]
It seems I don't have permissions to add labels to my own PRs in this project. Should I?

**Comment by robertmaynard** [comment]
> It seems I don't have permissions to add labels to my own PRs in this project. Should I?

You need to be in the correct owners group due to how GitHub permissions work.

**Comment by jameslamb** [comment]
Got it, thanks!

**Review by bdice — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The `rapids-cmake` module contains helpful general CMake functionality
 ### cpm
 
 The `rapids-cpm` module contains CPM functionality to allow projects to acquire dependencies consistently.
-For consistentcy All targets brought in via `rapids-cpm` are GLOBAL targets.
+For consistency, all targets brought in via `rapids-cpm` are GLOBAL targets.
 
 - `rapids_cpm_init()` handles initialization of the CPM module.
 - `raipds_cpm_find(<project> name BUILD_EXPORT_SET <name> INSTALL_EXPORT_SET <name>)` Will search for a module and fall back to installing via CPM. Offers support to track dependencies for easy package exporting
```
Comment (bdice) [inline README.md:72]: While we're here:
```suggestion
- `rapids_cpm_find(<project> name BUILD_EXPORT_SET <name> INSTALL_EXPORT_SET <name>)` Will search for a module and fall back to installing via CPM. Offers support to track dependencies for easy package exporting
```

**Review by bdice — APPROVED** [review]

**Pull request merged**